doc/module-system/module-system.chapter.md

```diff
@@ -115,3 +115,12 @@ A nominal type marker, always `"configuration"`.
 #### `class` {#module-system-lib-evalModules-return-value-_configurationClass}
 
 The [`class` argument](#module-system-lib-evalModules-param-class).
+
+#### `graph` {#module-system-lib-evalModules-return-value-graph}
+
+Represents all the modules that took part in the evaluation.
+It is a list of `ModuleGraph` where `ModuleGraph` has the following attributes:
+
+- `key`: `string`
```

Comment:
`file` seems useful for troubleshooting and should also be in the module attrset.
This is not implemented yet.

```suggestion
- `key`: `string` for the purpose of module deduplication
- `file`: `string` for the purpose of error messages and warnings
```